rest-api-spec/src/main/resources/rest-api-spec/test/cat.thread_pool/10_basic.yml

```diff
@@ -1,3 +1,30 @@
+"Test cat thread_pool total_wait_time output":
+  - skip:
+      version: " - 3.0.0"
```

Comment:
This was supposed to be `2.99.99` as well....will fix this in the backport/bwc PRs  🤦‍♂️